relay-server/src/services/project_cache.rs

```diff
@@ -1489,6 +1483,11 @@ impl Service for ProjectCacheService {
                             broker.handle_periodic_unspool()
                         })
                     }
+                    Some(message) = envelopes_rx.recv() => {
+                        metric!(timer(RelayTimers::ProjectCacheTaskDuration), task = "handle_envelope", {
```

Comment:
Could you also log a metric for the current size of the bounded queue somewhere (e.g. in the ticker)?